infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/project/OpenShiftWorkspaceServiceAccount.java

```diff
@@ -167,6 +168,7 @@ private OpenshiftRoleBinding createExecRoleBinding() {
         .endMetadata()
         .withNewRoleRef()
         .withName("exec")
+        .withNamespace(projectName)
```

Comment:
the fix is not related to the original issue, right?

Reply:
no, but it seems to fix https://github.com/eclipse/che/issues/16534

Reply:
@skabashnyuk FYI^

Reply:
CC @metlos

Reply:
it didn't help match in my case. @metlos can you try it from your side?

Reply:
Confirmed to fix #16534.

Reply:
looks like addressed by https://github.com/eclipse/che/pull/16619